### ISSUE TITLE
[UX][SEO] Better OOTB URL aliases when title contains forward slashes.

### ISSUE THREAD

**Issue by klonos** [backdrop/backdrop-issues#4837]
I've just created a blog post in our main site, with its title being "#states AND/OR/XOR conditionals explained". That resulted in a path of `news/states-andorxor-conditionals-explained`, which I had to then _manually_ update to the more SEO-friendly `news/states-and-or-xor-conditionals-explained`.

We should have that (replacing forward slashes with dashes) be the default OOTB for URL aliases.

**Comment by BWPanda** [comment]
I agree that your manual alias is better than the default one. I wonder if there's a reason/use-case for the current setting though...

**Comment by BWPanda** [comment]
Here's a PR: https://github.com/backdrop/backdrop/pull/4127

I suspect some of the other punctuation should be replaced too (like `_`, `|`, etc.), but will leave those for a follow-up.

**Comment by BWPanda** [comment]
And FTR, this change should be backwards-compatible as it'll only affect new sites.
